packaging/installer/netdata-updater.sh

```diff
@@ -254,7 +256,8 @@ if [ "${IS_NETDATA_STATIC_BINARY}" == "yes" ]; then
   fi
 
   # Do not pass any options other than the accept, for now
-  if sh "${TMPDIR}/netdata-latest.gz.run" --accept -- "${REINSTALL_OPTIONS}"; then
+  read -r -a reinstall_options <<< "$REINSTALL_OPTIONS"
+  if sh "${TMPDIR}/netdata-latest.gz.run" --accept -- "${reinstall_options[@]}"; then
```

Comment:
I think it's simpler to:

```suggestion
  if sh "${TMPDIR}/netdata-latest.gz.run" --accept -- ${REINSTALL_OPTIONS}; then
```

and add a comment to silence shellcheck.

Reply:
I _could_ have sworn I tested this and it fails in a similar way. Happy to re-test though :) And I'll report back...

Reply:
I'm slightly embarrased but you are right. Not becuase you are right becuase I'm aware of what happened and its all a bit silly :(